packages/lesswrong/components/vulcan-forms/FormWrapper.tsx

```diff
@@ -164,20 +164,21 @@ const FormWrapperNew = (props: WrappedSmartFormProps&{schema: any}) => {
  * Wrapper around an 'edit' form, which adds updateMutation. Should be used only
  * via FormWrapper.
  */
-const FormWrapperEdit = (props: WrappedSmartFormProps&{schema: any}) => {
+const FormWrapperEdit = <T extends CollectionNameString>(props: WrappedSmartFormProps<T>&{schema: any}) => {
   const currentUser = useCurrentUser();
   const collection = getCollection(props.collectionName);
   const { queryFragment, mutationFragment } = getFragments("edit", props);
-  const { extraVariables = {}, extraVariablesValues } = props
+  const { extraVariables = {}, extraVariablesValues = {} } = props
   
   const selector: DocumentIdOrSlug = props.documentId
     ? {documentId: props.documentId}
     : {slug: props.slug}
-  const { document, loading } = useSingle({
+  const { document, loading } = useSingle<AnyBecauseHard>({
     ...selector,
     collectionName: props.collectionName,
     fragment: queryFragment,
     extraVariables,
+    extraVariablesValues,
```

Comment:
This was missing, such that the one or two places where we attempted to pass in `version: draft` via `WrappedSmartForm` just didn't do anything.